src/commands/edge_app.rs

```diff
@@ -95,6 +95,52 @@ impl EdgeAppCommand {
         Ok(())
     }
 
+    pub fn create_in_place(&self, name: &str, path: &Path) -> Result<(), CommandError> {
+        let parent_dir_path = path.parent().ok_or(CommandError::FileSystemError(
+            "Can not obtain edge app root directory.".to_owned(),
+        ))?;
+        let index_html_path = parent_dir_path.join("index.html");
+
+        if !(Path::new(&path).exists() && Path::new(&index_html_path).exists()) {
+            return Err(CommandError::FileSystemError(format!(
+                "The directory {} should contain screenly.yml and index.html files",
+                parent_dir_path.display()
+            )));
+        }
+
+        let data = fs::read_to_string(path)?;
+        let mut manifest: EdgeAppManifest = serde_yaml::from_str(&data)?;
+
+        if !manifest.app_id.is_empty() {
+            return Err(CommandError::FileSystemError("app_id in screenly.yml should not be set".to_string()));
```

Comment:
Error is confusing though. Because if will try to remove it - still won't work. Also it's not a file system error